src/Boot/DrupalBoot8.php

```diff
@@ -36,23 +36,23 @@ class DrupalBoot8 extends DrupalBoot implements AutoloaderAwareInterface
     /**
      * @return \Symfony\Component\HttpFoundation\Request
      */
-    public function getRequest()
+    public function getRequest(): \Symfony\Component\HttpFoundation\Request
```

Comment:
When we add a return hint like this, we should remove the Doxygen when the doxygen has no description (e.g. this method). Is that possible?

Reply:
Let me poke at it. The "fix" for your initial request was to comment out the calls to remove docblock tags inside Rector, which is a very brute-force approach.

To be smahhter, I think we'll need to override these steps with our own, or contribute some patches upstream.

I'm ok to work on the longer steps if you think it's a worthwhile investment for Drush.

Reply:
Yeah, I think its worthwhile ... Would be great if you could put in a few hours and see how far you get. Thanks!

Reply:
If you arent available for more work on this, I'm OK with merging what we have.